src/components/PersonalData.js

```diff
@@ -15,13 +15,23 @@ import "../login/styles/index.scss";
 
 /* FORM */
 
-const validate = (values) => {
+const validatePersonalData = (values) => {
   const errors = {};
+  const spacePattern = /^\s+$/;
+  const onlyLettersPattern = /^[a-zA-ZäöåÄÖÅ]+(\s+[a-zA-ZäöåÄÖÅ]+)*$/;
```

Comment:
what do you think about reversing this regex (to not include integers and special characters), in case we add another language and don't want to add in all their unique characters?

Reply:
I have added all alphabets so any nationality can use their native spelling.

Reply:
We shouldn't do that (I was just trying to use a real example of why I thought matching for stuff we **don't** want seems more logical than matching for all things we want - i.e. all languages in the world). 

I was thinking something more like this for the regex (should match numbers and special characters that usually don't go in a name): [\d+\^\*\|\"\:\<\>[\]\{\}\`\\\(\)\'\%\;\#\@\&\$]+ 
(please check that it works and add/remove any special characters - don't forget to put a `\` in front of each character you add)